src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/segmentgroup/SpecimenDetails.java

```diff
@@ -41,7 +41,7 @@
  * &gt; {@link InvestigationSubject}
  */
 @SuppressWarnings("OptionalUsedAsFieldOrParameterType")
-public class Specimen extends SegmentGroup {
+public class SpecimenDetails extends SegmentGroup {
```

Comment:
Can we make this singular?

```suggestion
public class SpecimenDetail extends SegmentGroup {
```

Reply:
The others are `...Details` - so this would be for none or all of them

Reply:
Discussion on Teams: outcome was that `SpecimenDetails` was most favourable / least objectionable.

Can always be revisited later, but I'd like to get _some_ difference in place for the Specimen mapping